.gitignore

```diff
@@ -1,8 +1,6 @@
 
-# Created by https://www.gitignore.io/api/django,jekyll,visualstudiocode,pycharm
-
-### gitignore your environment
-env/
+# Created by https://www.gitignore.io/api/node,django,jekyll,pycharm,visualstudiocode
+# Edit at https://www.gitignore.io/?templates=node,django,jekyll,pycharm,visualstudiocode
```

Comment:
For me at least, it's missing `vim,linux` and change `pycharm` to `pycharm+all`. May be `macos` should be here too.